xep-0482.xml

```diff
@@ -204,4 +212,75 @@
     <p>This document requires no interaction with &REGISTRAR;.</p>
   </section1>
 
+  <section1 topic='XML Schema' anchor='schema'>
+    <code><![CDATA[<?xml version='1.0' encoding='UTF-8'?>
+
+<xs:schema
+    xmlns:xs='http://www.w3.org/2001/XMLSchema'
+    xmlns='urn:xmpp:call-invites:0'
+    targetNamespace='urn:xmpp:call-invites:0'
+    elementFormDefault='qualified'>
+
+  <xs:annotation>
+    <xs:documentation>
+      The protocol documented by this schema is defined in
+      XEP-0482: https://xmpp.org/extensions/xep-0482.html
+    </xs:documentation>
+  </xs:annotation>
+
+  <xs:element name='invite'>
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element ref='jingle'/>
+        <xs:element ref='external' minOccurs='0' maxOccurs='unbounded'/>
```

Comment:
Maybe add explicit `<xs:any namespace='##other' />` here to indicate that this is explicitly intended as an extension point.